sdk/datafactory/azure-mgmt-datafactory/HISTORY.rst

```diff
@@ -3,6 +3,156 @@
 Release History
 ===============
 
+0.8.0 (2019-08-07)
```

Comment:
Given there are SDK breaking changes in this release, shouldn't the major version change?

Reply:
Not necessarily (https://github.com/Azure/azure-sdk-for-python/blob/master/doc/dev/mgmt/mgmt_release.md - pls check bottom of the page).
0.7.0 also had breaking changes.
Unless you want to specifically change to 1.0.0.

Reply:
It's fine, let's re;ease with 0.8.0. Thanks!